pages/common/mkdir.md

```diff
@@ -1,16 +1,16 @@
 # mkdir
 
-> Creates a directory.
+> Create directories and set their permissions.
 > More information: <https://www.gnu.org/software/coreutils/mkdir>.
 
-- Create a directory in current directory or given path:
+- Create specific directories:
 
-`mkdir {{directory}}`
+`mkdir {{path/to/directory1 path/to/directory2 ...}}`
 
-- Create multiple directories in the current directory:
+- Create specific nested directories recursively:
```

Comment:
`nested` and `recursively` at the same time not sounds good for me. Based on the man page for `mkdir`:
```suggestion
- Create directories and their parents if needed:
```